CodeMaid/UI/Dialogs/Options/General/FeatureDataTemplate.xaml

```diff
@@ -0,0 +1,55 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:local="clr-namespace:SteveCadwallader.CodeMaid.UI.Dialogs.Options.General">
+    <DataTemplate DataType="{x:Type local:FeatureViewModel}">
+        <StackPanel>
+            <GroupBox Header="Cleanup">
```

Comment:
I think perhaps we should reorganize these settings.  One way would be to line them up with the Options (e.g. Cleaning, Collapsing, Digging, Finding, Formatting, Reorganizing, Switching, etc.) but they would be pretty fine grained group boxes.  Another way would be to line them up by where they are found in the UI (e.g. Main Menu, Context Menus, Tool Windows).  Thoughts?

Reply:
OK, I'll take the first suggestion, because a feature can have multiple menus in different context (main menus, context menus...), the second one might not work.